docs/_blog/2020-05-14-codewind-in-intellij-may.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "Codewind in IntelliJ IDEA"
+categories: blog
+author_picture: images/blog/author_icon_john_pitman.png
+author_url:
+author: John Pitman
+blog_description: "Update on support for Codewind in IntelliJ IDEA"
+permalink: codewind-in-intellij-may
+duration: 2 minutes
+tags: [IntelliJ]
+---
+
+Over the past few months we've been hard at work creating a plugin for IntelliJ IDEA to add support for Codewind. This is a tech preview, because there are still some fundamental features to be implemented, but we have made significant progress.
+
+Our first tech-preview release included basic support for Codewind, and last month we added the ability to show the log files, open the performance dashboard, and debug projects.
```

Comment:
We have intro blog that you can link if you'd like ---
[first tech-preview release](https://www.eclipse.org/codewind/introducing-eclipse-codewind-on-intellij.html)

Reply:
Changed